src/operation.ts

```diff
@@ -78,8 +78,8 @@ export default class Operation {
   }
 
   getSummary(): string {
-    if (this.schema?.summary) {
-      return this.schema.summary.trim();
+    if (this.schema?.summary && typeof this.schema.summary === 'string') {
+      return this.schema.summary;
     } else if (this.api.paths[this.path].summary) {
```

Comment:
Want to add this `typeof` logic to these other blocks? These ones handle callback objects.